File: AT01/C/Q01.c
#include <stdio.h>

int main(){

    float x, lucro;
    
    scanf("%f", &x);
    if(x < 20){
        lucro = x * 0.45;
        printf("%.2f", lucro+x);
    }else{
        lucro = x * 0.30;
        printf("%.2f", x+lucro);
    }
}

File: AT01/py/Q01.py
x = float(input())

def plucro():
    if x < 20:
        lucro  = x * 0.45
        print("{:.2f}".format(lucro+x))
    else:
        lucro = x * 0.30
        print("{:.2f}".format(x+lucro))

plucro()